src/puppetlabs/puppetdb/config.clj

```diff
@@ -149,7 +149,15 @@
      :temp-usage s/Int
      :memory-usage s/Int
      :max-command-size (pls/defaulted-maybe s/Int (default-max-command-size))
-     :reject-large-commands (pls/defaulted-maybe String "false")}))
+     :reject-large-commands (pls/defaulted-maybe String "false")
+
+     ;; The below config is intended for testing now and isn't documented
+     ;; or expected to be released. Perf testing will confirm if the
+     ;; parameters are needed and what their ideal value would be in our
+     ;; various deployment scenarios
+     :concurrent-writes (pls/defaulted-maybe s/Int 100)
+     :max-enqueued (pls/defaulted-maybe s/Int 1000000)
```

Comment:
What should we do about this one? I would rather not have a config item for it, but it seems like something we probably should.